_benchmark/user-guide/install-and-configure/configuring-benchmark.md

```diff
@@ -159,6 +160,46 @@ This section defines how OpenSearch versions are distributed.
 | :---- | :---- | :---- |
 | `release.cache` | Boolean | Determines whether newly released OpenSearch versions should be cached locally. |
 
+## Running OpenSearch Benchmark with AWS Signature Version 4
+
+OpenSearch Benchmark supports AWS Signature Version 4 authentication. To run Benchmark with Signature Version 4, use the following steps:
+
+1. Set up an [IAM user or an IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) and provide it access to the OpenSearch cluster using Signature Version 4 authentication.
+
+2. Set up the following environment variables for your IAM user:
+
+   ```bash
+   export OSB_AWS_ACCESS_KEY_ID=<IAM USER AWS ACCESS KEY ID>
+   export OSB_AWS_SECRET_ACCESS_KEY=<IAM USER AWS SECRET ACCESS KEY>
+   export OSB_REGION=<YOUR REGION>
+   export OSB_SERVICE=es
+   ```
+   {% include copy.html %}
+
+   If you want to set up an IAM role instead of an IAM user, use the following environment variables instead:
```

Comment:
These steps are a little off. On line 167 we tell the user to create either an IAM user or role and provide a link to instructions, line 169 assumes that the user has chosen to create an IAM user, and then this line reintroduces the idea of an IAM role. Please restructure for clarity.